Dockerfile

```diff
@@ -31,6 +32,26 @@ COPY --from=builder /usr/local/bundle /usr/local/bundle
 COPY --from=builder ${ZAMMAD_DIR}/contrib/docker/docker-entrypoint.sh /
 RUN contrib/docker/setup.sh runner
 
+RUN if [ "$RAILS_ENV" = "development" ]; then \
+  apt-get update && \
```

Comment:
Let's perhaps move this to a script in `.dev/docker` as well, to keep this file shorter.

Reply:
There should probably be at least `./dev/docker/start.sh` and `stop.sh` as well. Maybe also a `restart.sh` or a `rebuild.sh`, but those could also be added in a follow-up step, if they are needed (eg. to run after dependency update).